integration_tests/dbt_project.yml

```diff
@@ -0,0 +1,122 @@
+config-version: 2
+
+name: 'zendesk_source_integration_tests'
+version: '0.1.2'
+
+require-dbt-version: [">=0.18.0", "<0.20.0"]
+profile: 'integration_tests'
+
+vars:
+  zendesk_schema: zendesk_source_integration_tests
```

Comment:
This was needed to be added in order for the conditional within `stg_zendesk__ticket_schedule_tmp` to properly work within circleci.